azure-governance-cloudwise/ServicePrincipal.ps1

```diff
@@ -0,0 +1,134 @@
+﻿cls
```

Comment:
ServicePrincipals were currently not supported creation through JSON templates, so we sougnt to keep this manual for now. Documented in the install guide images/DeploymentGuide_and_Configuration_Guide_CloudWise.pdf > 1.3